docs/reference/troubleshooting/exemptions-and-package-updates.md

```diff
@@ -0,0 +1,97 @@
+---
+title: Exemption and Package Updates (5m)
+
+sidebar:
+  order: 1
+---
+
+
+## Exemptions and Package Updates in the Cluster
+
+This guide provides steps to debug issues with `Exemptions` and `Packages` not being applied or updated in your Kubernetes cluster. Common symptoms include:
+- Changes to Exemptions or Packages are not reflected in the cluster.
+- Expected behavior in workloads remains unaffected.
+- Logs indicate potential Kubernetes Watch failures.
+
+Follow this guide to identify and resolve these issues.
+
+## Initial Checklist
+
+Before diving into detailed debugging, ensure the following:
+
+- **Verify Configuration**:
+   - Ensure that Exemptions and Packages are defined correctly in your manifests.
+   - Refer to the specification documents for correct schema and examples:
+     - [Packages Specification](/reference/configuration/custom%20resources/packages-v1alpha1-cr)
+     - [Exemptions Specification](/reference/configuration/custom%20resources/exemptions-v1alpha1-cr)
+
+- **Namespace for Exemptions**:
+   - Ensure Exemptions are applied in the `uds-policy-exemptions` namespace, unless you are using an [override](/reference/configuration/uds-configure-policy-exemptions).
+
+- **Cluster and Deployment Status**:
+   - Confirm the cluster and relevant controller deployments are running without errors: 
+      ```bash
+      kubectl get pods -n pepr-system
+      ```
+
+## Troubleshooting Kubernetes Watch
+
+Kubernetes Watch is a mechanism used to monitor resource changes in real-time. Failures in Watch can cause Exemptions and Package updates to not propagate.
+
+### Steps to Check Watch Logs
+
+1. **Identify the Controller Pod**:
+   - Check the logs of the controller managing Exemptions using the following command:
+     ```bash
+     kubectl logs -n pepr-system deploy/pepr-uds-core | grep "Processing exemption"
+     ```
+
+   - If the logs **do not show entries similar to the following**, it may indicate that the Watch missed the event:
+     ```json
+     {"...":"...", "msg":"Processing exemption nvidia-gpu-operator, watch phase: MODIFIED"}
+     ```
+
+2. **Verify Package Processing**:
+   - Use the following command to check logs for Package processing:
+     ```bash
+     kubectl logs -n pepr-system deploy/pepr-uds-core-watcher -f | egrep "Processing Package"
```

Comment:
Was the `-f` (follow logs) on this one intentional? Command works just wanted to check!

Reply:
Yeah I left the `-f` thinking that the user may want to follow the logs while adding Packages or Exemptions to see in real time in another terminal. That is how I debug at least but it can come out if you prefer